locales/fr.json

```diff
@@ -355,13 +355,13 @@
 		},
 		"aria-roledescription": {
 			"pass": "aria-roledescription utilisé sur un élément sémantique supporté",
-			"incomplete": "Vérifier que la valeur de aria-roledescription est annoncée par les lecteurs d'écran supportés",
+			"incomplete": "Vérifier que la valeur d'aria-roledescription est annoncée par les lecteurs d'écran supportés",
```

Comment:
There's a few `'` instead of the curly `’`. I'm not sure if that matters to you but just noticed a few places where they were in. Up to you if you'd like to update them but we can merge as is.

Reply:
Indeed I forgot a lot of them 😅 I fixed this in the following commit

Reply:
Please use an apostrophe (’) rather than a prime ('). I have strong feelings about this 😅

Reply:
@stephenmathieson i did it in the commit ca645dc. I double checked, normally I did not miss any.